### DISCUSSION TITLE
Using "youtube API key" in Ytdlnis config

### DISCUSSION THREAD

**Discussion by sebadamus** [deniscerri/ytdlnis#538] [General]
Hi, 

I am new Ytdnlnis user, find it the best of the best :-) tried several other apps before... this one is just perfect.

I tried to configure Youtube API key, I create it from the "console.cloud.google.com" project setup, gave it all permissions over youtube API, and got the .JSON file, there I took the "client_secret" string and paste it in Ytdlnis youtubne API key.  Restarte the app but have no home screen suggestions as I had with the default program behaviour.

What could I have done wrong?

Thanks

**Comment by zaednasr** [comment]
Turn on home suggestions in the general seggings

**Comment by error-reporting** [comment]
how to configure youtube api key and gave it all permissions over youtube API, and then got the .JSON file

**Reply by deniscerri** [reply]
u dont need json file, you need that api key which is a long gibberish word
something like this
Altsrz-STsgfjsh-dsjhgdsghjds-sdgfsjgsdhgsdhgsg

**Reply by deniscerri** [reply]
Clear the results in the home screen

**Reply by deniscerri** [reply]
Also, delete this message, dont share the key in the open like this
